subprojects/common/junit-utils/src/main/java/com/avito/android/util/SerializeUtils.kt

```diff
@@ -10,6 +10,8 @@ import org.junit.Assert.assertEquals
 
 object SerializeUtils {
 
+    val isShort: Int = 11
```

Comment:
`isX` should return bool rather than Int

Reply:
debug leftover, removed